src/allmydata/web/magic_folder.py

```diff
@@ -34,6 +34,9 @@ def post_json(self, req):
             for (status, ts) in item.status_history():
                 d[status + '_at'] = ts
             d['percent_done'] = item.progress.progress
+            if item.progress.progress > 100.0:
+                print("TOO MUCH UPLOAD PROGRESS:")
+                print(json.dumps(item))
```

Comment:
Need to do something about these prints.  I guess this is debug information for a different bug fix.